### ISSUE TITLE
Wrong ram manufacturer 

### ISSUE THREAD

**Issue by ferhadquluzade** [sebhildebrandt/systeminformation#936]
**Describe the bug**
My device is M1 Mac and ram manufacturer is Micron, but the systeminformation package shows  `Apple`.
**To Reproduce**
Running the code
```js
import si from "systeminformation";

const rams = await si.memLayout();
var result = ``;
rams.forEach((ram, index) => {
  result += `\nram#${index + 1}:
\tmanufacturer: ${ram.manufacturer}
`;
});

console.log(result);
```

**Current Output**
Manufacturer: Apple

**Expected behavior**
Manufacturer: Micron

**Environment**
 - systeminformation package version: 5.23.5
 - OS: macOS 15
 - Hardware: MacBook Air M1 

**Additional context**
I got expected behavior from built-in tool called  "System information"

**Comment by sebhildebrandt** [comment]
@ferhadquluzade ... will provide a new version tomorrow (right now I am testing the fix ... looks fine so far)

**Comment by sebhildebrandt** [comment]
@ferhadquluzade fixed, version 5.23.22 just released. Can you test it on your side?
